src/header/js/header-shopping-cart.js

```diff
@@ -0,0 +1,70 @@
+import {$assign as $, $dispatch, $replaceAll, $renderSvgOrImg} from '../../shared/js/shared';
+import {$cart} from '../../shared/js/iconPaths';
+
+const prefix = 'esri-header-shopping-cart';
+
+export default () => {
+	const $target = $('div', {class: prefix});
+
+	$target.addEventListener('click', (event) => {
+		$dispatch($target, 'header:click:shoppingCart', {event});
+	});
+
+	/* Shopping Cart: Control
+	/* ====================================================================== */
+
+	const $cartItems = $('div',
+		{class: `${prefix}--items`, id: `${prefix}--items`}
+	);
+
+	$target.addEventListener('header:update:cart', ({detail}) => {
+		const $control = $('div',
```

Comment:
I don't think `$control` needs to be redefined every time the cart gets updated. It can probably be set once outside the event listener callback.